pages/c/_cluster/explorer/Glance.vue

```diff
@@ -0,0 +1,81 @@
+<script>
+import GradientBox from '@/components/GradientBox';
+
+export default {
+  components: { GradientBox },
+  props:      {
+    provider: {
+      type:     String,
+      required: true
+    },
+    kubernetesVersion: {
+      type:     String,
+      required: true
+    },
+    totalNodes: {
+      type:     Number,
+      required: true
+    },
+    created: {
+      type:     String,
+      required: true
+    }
+  }
+};
+</script>
+
+<template>
+  <GradientBox class="glance" primary-color="rgba(61, 152, 211, 1)" :show-tab="false">
+    <div class="tile">
+      <h1>{{ provider }}</h1>
+      <label>{{ t('glance.provider') }}</label>
+    </div>
+    <div class="tile">
+      <h1>{{ kubernetesVersion }}</h1>
+      <label>{{ t('glance.version') }}</label>
+    </div>
+    <div class="tile">
+      <h1>{{ totalNodes }}</h1>
+      <label>{{ t('glance.nodes.total.label') }}</label>
+    </div>
+    <div class="tile">
+      <h1><LiveDate :value="created" :add-suffix="true" :show-tooltip="false" /></h1>
+      <label>{{ t('glance.created') }}</label>
+    </div>
+  </GradientBox>
+</template>
+
+<style lang="scss">
+    .glance {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        padding: 11px;
+
+        .tile {
+            display: flex;
+            flex: 1;
+            flex-direction: column;
+            align-items: flex-start;
+            padding: 9px 0 3px 0;
+
+            &:not(:last-of-type) {
+                border-right: 1px solid rgba(61, 152, 211, .5);
```

Comment:
All the colors throughout should come from CSS variables defined in _light and _dark.

Reply:
Done.